src/test/java/io/vertx/test/codegen/testtype/EnumHolder.java

```diff
@@ -0,0 +1,16 @@
+package io.vertx.test.codegen.testtype;
+
+import io.vertx.test.codegen.testenum.ValidEnum;
+
+import java.util.concurrent.TimeUnit;
+
+/**
+ * @author <a href="mailto:julien@julienviet.com">Julien Viet</a>
+ */
+public interface EnumHolder {
+
+
+  ValidEnum apiEnum();
+  TimeUnit otherEnum();
```

Comment:
right, they shall be tested rather in CollectionHolder, I will add test (I added enum test after Collection tests)